apisix/init.lua

```diff
@@ -297,8 +297,8 @@ function _M.http_access_phase()
     router.router_http.match(api_ctx)
 
     -- run global rule
-    plugin.run_global_rules(api_ctx, router.global_rules, "access")
     api_ctx.global_rules = router.global_rules
```

Comment:
Do it in run_global_rules?
Need to update the api_router.lua too

Reply:
@spacewander  
Thanks for reviewing.
I still have some doubts. The original code also assign value to `api_ctx.global_rules` after `run_plugin`, but it does not cause a bug like this. .

Reply:
> @spacewander
> Thanks for reviewing.
> I still have some doubts. The original code also assign value to `api_ctx.global_rules` after `run_plugin`, but it does not cause a bug like this. .

I re-checked the original code, it filtered plugins of global rule and assigned them to `api_ctx.plugins`, that is why..